src/codeu/chat/client/commandline/Chat.java

```diff
@@ -37,6 +37,18 @@
 
   public Chat(Context context) {
     this.panels.push(createRootPanel(context));
+      panel.register(“info”, new Panel.Command() {
+        @Override
+        public void invoke(Scanner args) {
+            final ServerInfo info = context.getInfo();
+            if (info == null) {
+                System.out.println("The server did not send us a valid info object")
+                } else {
+                // Print the server info to the user in a pretty way
```

Comment:
Now that you've merged in the changes from master, you can just delete this block of code. Make sure to still add your own code that prints the uptime information to the info command below!